app/controllers/api/v1/documents.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'doorkeeper/grape/helpers'
+
+module API
+  module V1
+    class Documents < Grape::API
+      format :json
+
+      helpers Doorkeeper::Grape::Helpers
+
+      before do
+        def document
+          @document = Account.find_by_uid(params[:uid]).profile.documents
+        end
+      end
+
+      desc 'Documents related routes'
+      resource :document do
+        desc 'Creates new document'
+        params do
+          requires :profile_id, type: String, desc: 'Id of profile'
+          requires :upload,     type: File,   desc: 'Document upload'
+          requires :doc_type,   type: String, desc: 'Type of document'
+          requires :doc_number, type: String, desc: 'Number of document'
+          requires :doc_expire, type: String, desc: 'Expiry date of document'
+        end
+        post '/create' do
```

Comment:
don't save space, use multiple lines of code, if validation is incorrect what is the error code it will return ?
how do you know error are properly managed ?